.github/workflows/build.yaml

```diff
@@ -50,11 +50,8 @@ jobs:
         # Can be changed back to npm@7 when https://github.com/npm/cli/issues/3637 is resolved
         run: npm install -g npm@7.20.2
 
-      - name: Install verdaccio
-        run: npm install -g verdaccio
-
       - name: Install Dependencies
-        run: npm install --force
+        run: npm run bootstrap-ci
```

Comment:
Sorry, I think I did a wrong suggest here, we don't have `bootstrap-ci` script, only `bootstrap-evergreen` (that we probably should rename) 🙈